CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `mypy` rule expecting explicit `override` markers for method overrides
```

Comment:
wonder whether a tiny note in the contributing guidelines makes sense as not eveyrone might know that and understand it based on the mypy error